index.html

```diff
@@ -40,13 +40,274 @@ <h3 class="large-text"> I can help you build a product, feature or website. Look
+                  <h4 class="language-text">html</h4>
+                </div>
+              </div>
+              <br>
+              <div class="see-container">
+                <a class="see-project">See Project</a>
+              </div>
+            </div>
+          </div>
+          <br>        
+          <div class="card">
+            <div class="cardlet">
+              <h3 class="project-title">Multi-Post Stories<br>Gain+Glory</h3>
+              <div class="languages-container">
+                <div class="language-box" id="ruby">
+                  <h4 class="language-text">Ruby on rails</h4>
+                </div>
+                <div class="language-box" id="css">
+                  <h4 class="language-text">css</h4>
+                </div>
+                <div class="language-box" id="js">
+                  <h4 class="language-text">JavScript</h4>
+                </div>
+                <div class="language-box" id="html">
+                  <h4 class="language-text">html</h4>
+                </div>
+              </div>
+              <br>
+              <div class="see-container">
+                <a class="see-project">See Project</a>
+              </div>
+            </div>
+          </div>
+          <br>
+        </div>
+      </section>
+      <br>
+      <img src="images/illustration 1- about me-mobile@2x.png" id="paper-bird-1" alt="decorative image">
+      <br>
+      <section class="about-myself">
+        <h2 class="about-me">About Me</h2>
+        <div class="large-text-container"></div>
+        <h3 class="large-text-2"> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hesitate to contact me.</h3>
+        <div class="image-div">
+          <img src="images/illustration 2-about me mobile@2x.png" id="paper-bird-2" alt="decorative image">
+        </div>
+        <div class="get-container">
+          <a class="get-resume">Get My Resume</a>
+        </div>
+      </section>
+      <br>
+      <br>
+      <section class="skills">
+        <div class="card-container">
+          <div class="skill-card">
+            <div class="icon-container"><img src="images/languages-icon.png" id="language-img" alt="languages icon"></div>
+            <br>
+            <br>
+            <div class="skill-text-container">
+              <h3 class="skill-title">Languages</h3>
+            </div>
+            <div class="skill-container">
+              <div class="skill-box">
+                <h4 class="skill-text">JavaScript</h4>
+              </div>
+              <div class="skill-box">
+                <h4 class="skill-text">Ruby</h4>
+              </div>
+              <div class="skill-box">
+                <h4 class="skill-text">HTML</h4>
+              </div>
+              <div class="skill-box">
+                <h4 class="skill-text">CSS</h4>
+              </div>
+            </div>
```

Comment:
- [ ] I think you could use the `<ul>` (unordered list) tag instead of the `<div>` tag because it's the common practice when listing items. Remember that according to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), we try to use the appropriate HTML5 semantic tags (e.g., section, header, nav, ul, li) over `<div>`s. :raised_hands: